internal/clientconn/conn.go

```diff
@@ -441,6 +442,24 @@ func (c *conn) route(connCtx context.Context, reqHeader *wire.MsgHeader, reqBody
 
 		command = document.Command()
 
+		if comment, cerr := document.Get("comment"); cerr == nil {
+			if cmt, ok := comment.(string); ok {
+				var spanCtx trace.SpanContext
+
+				spanCtx, cerr = spanContextFromComment(cmt)
+				if cerr != nil {
+					c.l.DebugContext(connCtx, "Failed to extract span context from comment", logging.Error(err))
+				}
+
+				connCtx = trace.ContextWithRemoteSpanContext(connCtx, spanCtx)
+
+				var remoteCtxSpan trace.Span
+				connCtx, remoteCtxSpan = otel.Tracer("").Start(connCtx, command)
```

Comment:
Let's create an extra span only if span info was given, not for any string comment

Reply:
Is there any particular reason why we should do it?
I think, it would be nice to have a parent span somewhere, and this place is a good candidate.